### PR TITLE
Visor STEP + 1/8" CNC Mod + Revision # + date fix

### PR DESCRIPTION
**Pull request by silvxm**
- Uploaded Visor STEP
- Added DXF and STEP of shield mount for clearance to cut with 1/8" EM, easier for manufacture
- added revision numbers
- fixed date on visor drawing

### CODE REVIEW

**Comment by engnadeau** [comment]
@silvxm please update the README to mention these new CNC files

**Comment by engnadeau** [comment]
@silvxm i'm looking at the CNC models, how does the 1/8" EM fit in the inner corners?

**Comment by engnadeau** [comment]
@silvxm i just added a `CHANGELOG.md` in a recent PR, can you please update your branch with respect to master and add these changes to the changelog

**Comment by silvxm** [comment]
> @silvxm i'm looking at the CNC models, how does the 1/8" EM fit in the inner corners?

Sure does, MT concept milled it. Checked clearances beforehand

**Comment by silvxm** [comment]
@nnadeau I edited changelog, readme. ready for review

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,3 +6,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Calendar Versioning](https://calver.org/).
 
 ## Unreleased
+
+## (20.04.01) - 2020-04-07
```
Comment (engnadeau) [inline CHANGELOG.md]:
```suggestion
```

hasn't been "released" yet, so these changes are still under the unreleased section

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,3 +6,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+
+### Added
+
+- COVID-001-Face-Shield-Frame_CNC_1_8-R01 .STEP and DXF for CNC milling w/ 1/8" cutter.
```
Comment (engnadeau) [inline CHANGELOG.md]:
```suggestion
- `COVID-001-Face-Shield-Frame_CNC_1_8-R01` STEP and DXF for CNC milling w/ 1/8" cutter
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,3 +6,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+
+### Changed
+
+- Revision numbers on all model files
+- Added Descriptor of CNC file addition to README.md
```
Comment (engnadeau) [inline CHANGELOG.md]:
```suggestion
- Added descriptor of CNC file addition to `README.md`
```

File: README.md [hunk 4]
```diff
@@ -60,8 +60,9 @@ While the design is nearly identical, we have made the following modifications:
 - Frame
   - -0.25mm horizontal compensation
   - Thickness of 1/8" (3.175mm)
+  - A second iteration of the frame is available and accommodates CNC milling with a 1/8" cutter.
```
Comment (engnadeau) [inline README.md]:
```suggestion
  - A [second iteration of the frame](models/COVID-001-Face-Shield-Frame_CNC_1_8-R01.STEP) is available and accommodates CNC milling with a 1/8" cutter
```

**Review by engnadeau — CHANGES_REQUESTED** [review]

**Comment by silvxm** [comment]
@nnadeau changes committed!

**Review by engnadeau — APPROVED** [review]

**Pull request merged**